test/test15.c

```diff
@@ -1,86 +1,43 @@
 #include <stdio.h>
 #include <stdlib.h>
-#include <string.h>
+#include <unistd.h>
 #include "../Annotations/Annotations.h"
```

Comment:
This test seems to have been completely changed. Can you explain the reasoning for that, rather than deleting this test and creating a new one (which would have a cleaner diff).

Reply:
That was a mistake on my part. I'll add it back as test18